R/flea_starters.R

```diff
@@ -68,3 +68,84 @@ ff_starters.flea_conn <- function(conn, week = 1:17, ...) {
 
   return(x)
 }
+
+####  Flea ff_starters Detail ####
+
+#' Get detail of starters and bench
+#'
+#' @param conn the list object created by `ff_connect()`
+#' @param week a numeric or numeric vector
+#' @param ... other arguments (currently unused)
+#'
+#' @describeIn ff_starters Fleaflicker Detail: returns a detailed version of flea_starters with injuries and no filters
+#'
+#' @examples
+#' \donttest{
+#' try({ # try only shown here because sometimes CRAN checks are weird
+#'   conn <- fleaflicker_connect(season = 2020, league_id = 206154)
+#'   ff_starters_detail.flea_conn(conn)
+#' }) # end try
+#' }
+ff_starters_detail.flea_conn <- function(conn, week = 1:17, ...) {
```

Comment:
> Pulling injury from the ff_starters which uses the box score is the way to go for injury designation it seems. ff_starters has the correct data but it isn't being pulled in which is what I did and removed the filter for players_id NA. I think having that unfiltered gives better context to who played and what the score was because it show the slots left open.
>
>I see the value of how you all have it, so it may warrant both or people can just add that filter themselves....lol. I think the filter, the injury fields, and adding BENCH to the group column is the only major changes.

I think I agree with your thought process overall and would include it in the main ff_starters logic, rather than as a second function. Can you modify the main ff_starters function instead?